Design.md

```diff
@@ -84,9 +84,12 @@ The release process integrates with Fedora's release milestones in the following
     - The `next` stream switches to weekly releases to closely track the GA content set.
 - Fedora General Availability
     - Fedora CoreOS re-orients its release schedule in the following way:
-        - Week 0 (GA release): triple release;`next` with latest Fedora N content
-        - Week 1: triple release; `testing` release promoted from previous `next`
-        - Week 3: triple release; `stable` release promoted from previous `testing`, now fully rebased to Fedora N. `next` and `testing` are now in sync.
+        - Week 0 (GA release): triple release:
+          - `testing` release promoted from previous `next`
+          - `next` release contains latest Fedora N content
+        - Week 2: triple release:
+          - `stable` release promoted from previous `testing`, now fully rebased to Fedora N
+          - `testing` and `next` are now in sync
```

Comment:
I feel like with this change we should probably ship out a `next` as soon as GA content is finalized:

```suggestion
        - Week -1 (Fedora "Go" Decision): `next` release:
          - `next` release ASAP with final Fedora GA content after "Go" decision
        - Week 0 (GA release): triple release:
          - `testing` release promoted from previous `next`
          - `next` release contains latest Fedora N content, including bodhi updates
        - Week 2: triple release:
          - `stable` release promoted from previous `testing`, now fully rebased to Fedora N
          - `testing` and `next` are now in sync
```

Reply:
I thought that was implied by

> The `next` stream switches to weekly releases to closely track the GA content set.

but I can add it in to be extra clear.